### ISSUE TITLE
Web - Develop Social Compliance System - Incorrect Page Title

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#81]
The page TITLE should be "Why Develop a Social Compliance System? - Comply Chain - United States Department of Labor" instead of only "Comply Chain".

You should tell assistive technology users like screen reader users and search engines what the page is followed the application and then organization (DOL) and you could even add in ILAB spelled out before the DOL part.

The page TITLE format applies to all Comply Chain pages, so please fix accordingly.

Develop Social Compliance System - https://dolcontentdev.opadev.dol.gov/ilab/complychain/build/why-develop

**Comment by KyleLivingstonDOL** [comment]
Hi John,

thanks for this. Do you mind explaining why every page title needs to have the format: "Title - Comply Chain - United States Department of Labor"? Would it be a 508 issue if we only had it on the homepage buttons?

Example, each button would be Title - Comply Chain - United States Department of Labor

**Comment by jfc3-dol** [comment]
In the HTML there is a code called a TITLE tag that tells the user what page they are the first thing. The first thing a screen reader does it read the TITLE of the page, number of links, and a few other things to the user before the content.

By using that format, you tell the user the page you are on then the application in this case followed by it's from DOL. When doing it for a regular ILAB page, for example, it should be "About Us - ILAB (Bureau of International Labor Affairs) - United States Department of Labor".

Doing all this helps the user first and then for SEO/search results.

**Comment by Abychalil** [comment]
title added in the requested format..

**Comment by jfc3-dol** [comment]
@Abychalil,

Thanks for correcting the screen TITLE. Will look at it when the next version/build is sent, and I  have time to test.

**Comment by jfc3-dol** [comment]
The page TITLE is now "Why Develop a Social Compliance System? - Comply Chain - United States Department of Labor", which doesn't tell the user which Step they are on, but it's much better than only being "Comply Chain".

So I'm going to close the issue.
